controller/api/fio.js

```diff
@@ -1121,22 +818,21 @@ class FIOCtrl {
             });
           }
         }
-      };
+      } else {
+        console.log(`${logPrefix}No domains to burn.`);
+      }
 
-      await processActions();
+      console.log(
+        `${logPrefix}Update FIO Block Number for burn NFTS: ${lastIrreversibleBlock}`,
```

Comment:
do we need backspace after prefix like in other places?

Reply:
Done